include/deal.II/grid/grid_tools_cache.h

```diff
@@ -150,6 +150,17 @@ namespace GridTools
     const Mapping<dim, spacedim> &
     get_mapping() const;
 
+
+    /**
+     * Return the cached covering_rtree object, constructed using
+     * bounding boxes to identify the processes owning
+     * each portion of the mesh in parallel::Triangulations .
+     * This a collective operation: it must be called by
+     * all processes at the same time.
```

Comment:
can you explain why? namely, because a call to this function implies a call to ... which is itself collective